Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -248,6 +272,20 @@ void MenuBar::AddTableColumnsMenu(QMenu* view_menu)
   }
 }
 
+void MenuBar::UpdateToolsMenu(bool emulation_started)
+{
+  const bool enable_wii_tools = !emulation_started || !Settings::Instance().IsWiiGameRunning();
+  m_perform_online_update_menu->setEnabled(enable_wii_tools);
+  if (enable_wii_tools)
+  {
+    IOS::HLE::Kernel ios;
+    const auto tmd = ios.GetES()->FindInstalledTMD(0x0000000100000002);
```

Comment:
Maybe we should put some enums/defines/constexpr/whatever somewhere in a common place for those "well-known" title ids? boot2 (which is ignored during the update), BC/MIOS (during boot) and IOS 80 (used by the system menu) would also be cancidates for that.

Actually, we do have some of those already in IOS.cpp, but they're local to the file (not shared with the outside world).